GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1990,19 +1990,12 @@ private void SetGitModule(object sender, GitModuleEventArgs e)
 
         private void TranslateToolStripMenuItemClick(object sender, EventArgs e)
         {
-            Process.Start("https://github.com/gitextensions/gitextensions/wiki/Translations");
+            OsShellUtil.OpenUrlInDefaultBrowser(@"https://github.com/gitextensions/gitextensions/wiki/Translations");
         }
 
         private void FileExplorerToolStripMenuItemClick(object sender, EventArgs e)
         {
-            try
```

Comment:
Any reasoning to remove this try-catch?
I am lost in the previous discussions...

Reply:
We ultimately want all exceptions to be routed via the `BugReporter`, which will present a user with a message about the error. One caveat - we want exceptions to be on the UI thread, else exceptions are considered fatal and will lead to the app closed.

So generally speaking the only `try..catch` scenario we should have in the code is catch an exception which we can 
- ignore, or
- wrap as a BugReporter-known exception and rethrow.